docs/config.js

```diff
@@ -0,0 +1,29 @@
+// Valid languages to server-side render in production
+const LANGUAGES = ['en'];
+
+// Server side rendered languages
+const LANGUAGES_SSR = ['en'];
+
+// Work in progress
+const LANGUAGES_IN_PROGRESS = LANGUAGES.slice();
+
+const LANGUAGES_IGNORE_PAGES = (pathname) => {
+  // We don't have the bandwidth like Qt to translate our blog posts
+  // https://www.qt.io/zh-cn/blog
+  if (pathname === '/blog' || pathname.startsWith('/blog/')) {
+    return true;
+  }
+
+  if (pathname === '/size-snapshot/') {
+    return true;
+  }
+
+  return false;
+};
+
+module.exports = {
```

Comment:
I notice this is cjs so it can be used in next.config.js bit also imported in application code as ESM. We could convert to [next.config.mjs](https://github.com/mui/mui-toolpad/pull/1404) and author this module in ESM

Reply:
I haven't yet converted the scripts that use it to ESM. It is indeed `import`ed in few places but only in TypeScript files that are transpiled to CJS. After I convert the scripts to ESM, I can update this config file as well.